### ISSUE TITLE
Download images from Issues

### ISSUE THREAD

**Issue by razvanursu** [benr77/obsidian-gitlab-issues#20]
Hi and thanks for creating this plugin!

I was thinking that images could also be displayed in the GitLab Issue. As I see it, the best option os to download the remote image locally to some "/images/" folder, where Obsidian can render it locally.

I have also thought of extending the GitLab local uploads link with the gitlab instance and project name and use the image as an external one. However, I guess this would require authentication, in order to see the image in the Obsidian Note, which makes everything more complicated.

What do you think about this? 😄

**Comment by benr77** [comment]
What images are you referring to? Ones that are embedded into issue descriptions and comments?

**Comment by razvanursu** [comment]
Yup, exactly! The ones referenced in the Markdown as ![Image Name](image/local/ref/img.png), for example.

**Comment by razvanursu** [comment]
Hi, any update on this issue? Should there be any help needed, please let me know, as I could perhaps contribute :)
